graphprot/Metrics.py

```diff
@@ -55,22 +111,19 @@ def __init__(self, prediction, y, target, threshold=4, binary=True):
         print('Threshold set to {}'.format(self.threshold))
 
         if self.binary == True:
-
-            prediction_bool = get_boolean(
+            prediction_binary = get_binary(
                 self.prediction, self.threshold, self.target)
-            y_bool = get_boolean(
+            y_binary = get_binary(
                 self.y, self.threshold, self.target)
             classes = [0, 1]
-
             false_positive, false_negative, true_positive, true_negative = get_comparison(
-                prediction_bool, y_bool, self.binary, classes=classes)
+                prediction_binary, y_binary, self.binary, classes=classes)
             
         else:
-            if self.target == 'class':
+            if self.target == 'capri_class':
                 classes = [1, 2, 3, 4, 5]
-            else:
+            if self.target == 'binclass':
```

Comment:
It's always better to have a default case (that potentially raise an error)
```python
if target == 'capri':
     classes = 
elif target == 'binclass':
      classes = 
else:
      raise ValueError('target must be capri on binclass')
``` 

so that if someday we by accident set target to `bin_class` instead of `binclass` we know what's happening

Reply:
It's also always better to be as consistent as possible in the names. So if we have `capri_class`  I would suggest we use `bin_class`.